### DISCUSSION TITLE
Cannot find FBA fee, Could you help?

### DISCUSSION THREAD

**Discussion by Javadebi** [amzn/selling-partner-api-models#3412] [General]
Hello I want to find my own products FBA fee. Right now I'm using `GET_FBA_ESTIMATED_FBA_FEES_TXT_DATA` but its old and sometimes gives empty. What should I call for more up to date FBA fee?

**Comment by ritikacd** [comment]
@Javadebi The content of FBA Fee Preview Report gets updated at least once every 72 hours. To successfully generate a report, specify the StartDate parameter for a minimum 72 hours prior to NOW and EndDate to NOW. There's no other report as of now to get the FBA fees per SKU, but you can review other resources in Seller Central on FBA features and fees: https://sellercentral.amazon.com/help/hub/reference/201074400

Hope this will help!
